prime-router/build.gradle.kts

```diff
@@ -927,6 +927,10 @@ dependencies {
 
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3")
 
+    // validations
+    implementation("com.networknt:json-schema-validator:1.4.0")
+    implementation("io.konform:konform-jvm:0.4.0")
```

Comment:
why do we need this library? doesn't `json-schema-validator` also provide facility to create validators for data contained in the yaml document?

Reply:
I chose this library for further validation of configuration values. Once the structure is correct and deserialized to an object we can take a closer look at whats in them and decide if its acceptable.

For example we can ensure that the following are correctly formatted:
```yaml
      jurisdictionalFilter:
        - "hasAtLeastOneOf(waters_submitter, sender_id)"
        - "orEquals(patient_state, GH, ordering_facility_state, GH)"
```

Reply:
You are right there are validations built into json schema but they are pretty primitive (min/max for numbers, regex for strings, maxLength for arrays, etc). It's enough for baseline validations but not for something more specific to Report Stream.